yt_dlp/extractor/goodgame.py

```diff
@@ -17,26 +18,26 @@ class GoodGameIE(InfoExtractor):
             'title': r're:Рейтинговые герои \d{4}-\d{2}-\d{2} \d{2}:\d{2}$',
             'description': 'md5:ca5faa69b5d9215afdc65caeca68e1f6',
             'thumbnail': r're:^https?://.*\.jpg$',
-            'is_live': True,
             'live_status': 'is_live',
             'view_count': int,
         },
         'params': {'skip_download': 'm3u8'},
+        'skip': 'May not be online',
     }]
 
     def _real_extract(self, url):
         channel_name = self._match_id(url)
-
         response = self._download_json(f'https://api2.goodgame.ru/v2/streams/{channel_name}', channel_name)
 
-        if response.get('status') != 'Live':
-            raise ExtractorError(f'{channel_name} is offline', expected=True)
+        formats, subtitles = [], {}
+        if response.get('status') == 'Live':
+            formats, subtitles = self._extract_m3u8_formats_and_subtitles(
+                f'https://hls.goodgame.ru/manifest/{response["id"]}_master.m3u8',
```

Comment:
@pukkandan Shouldn't we use `response.get('id)` here?

Reply:
why? If id is None, the request will go to `https://hls.goodgame.ru/manifest/None_master.m3u8` which will fail anyway